packages/cli/src/controllers/oauth/oAuth2Credential.controller.ts

```diff
@@ -123,16 +117,9 @@ export class OAuth2CredentialController extends AbstractOAuthController {
 				additionalData,
 			);
 
-			const token = new Csrf();
-			if (
-				decryptedDataOriginal.csrfSecret === undefined ||
-				!token.verify(decryptedDataOriginal.csrfSecret as string, state.token)
-			) {
+			if (this.verifyCsrfState(decryptedDataOriginal, state)) {
 				const errorMessage = 'The OAuth2 callback state is invalid!';
-				this.logger.debug(errorMessage, {
-					userId: req.user?.id,
```

Comment:
Out of curiosity: Why remove the user ID here?

Reply:
This part is from when I was trying to streamline the two controllers even more, but had to revert for now. I personally did not see the value in adding `userId` here considering that if/when this error happens, `userId` won't provide any valuable insight. The only argument I could come up for keeping this was that "well, someone added it, they must have had a reason".